src/consensus.rs

```diff
@@ -103,17 +112,21 @@ impl FromBytes for EraReport {
 #[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
 #[cfg_attr(test, derive(serde::Serialize, serde::Deserialize))]
 // See: https://github.com/casper-network/casper-node/blob/8ca9001dabba0dae95f92ad8c54eddd163200b5d/node/src/types/block.rs#L748-L753
-pub struct EraEnd {
+pub struct EraEndV1 {
```

Comment:
Why V1?

Reply:
We don't support V1 anywhere else, so I think we can drop support here and just do V2.